pkg/kfconfig/awsplugin/types.go

```diff
@@ -34,6 +33,8 @@ type AwsPluginSpec struct {
 
 	ManagedObjectStorage *ObjectStorageConfig `json:"managedObjectStorage,omitempty"`
 
+	EnableBasicAuth *bool `json:"enableBasicAuth,omitempty"`
```

Comment:
As we have a separate auth field, do we need this one? Can we infer the auth option based on `*Auth` value user sets?

Reply:
Yeah it makes sense, we can do so, let me fix.